packages/experiments/src/components/Announced/examples/Announced.Asynchronous.Example.tsx

```diff
@@ -0,0 +1,151 @@
+    super(props);
+    this.state = {
+      photos: this._createPhotos(),
+      total: 0,
+      seconds: 0,
+      announced: undefined,
+      percentComplete: 0,
+      loading: false
+    };
+
+    this._renderPhotos = this._renderPhotos.bind(this);
+    this._renderAnnounced = this._renderAnnounced.bind(this);
+    this._onFocusPhotoCell = this._onFocusPhotoCell.bind(this);
+    this._onToggleChange = this._onToggleChange.bind(this);
+
+    this.increaseTotal = setInterval(() => {
+      if (this.state.loading && this.state.total < this.state.photos.length) {
+        this.setState({ total: this.state.total + 1 });
+      }
+    }, 2000);
+
+    this.timer = setInterval(() => {
+      if (this.state.loading) {
+        this.setState({ seconds: this.state.seconds + 1 });
+
+        if (this.state.seconds % this.delay === 0) {
+          this.setState({
+            announced: (
+              <Announced message={`${this.state.total}/${this.state.photos.length} photos loaded`} id={'announced-' + this.state.total} />
+            )
+          });
+        } else {
+          this.setState({ announced: undefined });
+        }
+      }
+    }, 1000);
+
+    this.delay = 10;
+  }
+
+  public render(): JSX.Element {
+    const { percentComplete } = this.state;
+
+    return (
+      <VerticalStack gap={10}>
+        <div>
+          Turn on Narrator and check the toggle to start loading photos. Announced should announce the number of photos loaded every 10
+          seconds, as that is the delay chosen for this example.
+        </div>
+        <div>When focusing on a photo that hasn't loaded yet, the Announced component should announce "Photo loading".</div>
+        <Toggle label="Check to start loading photos" onText="Start/Resume" offText="Pause" onChange={this._onToggleChange} />
+        <ProgressIndicator label={percentComplete < 1 ? 'Loading photos' : 'Finished loading photos'} percentComplete={percentComplete} />
+        <FocusZone elementType="ul" className="ms-AnnouncedExamples-photoList">
+          {this._renderAnnounced()}
+          {this._renderPhotos()}
+        </FocusZone>
+      </VerticalStack>
+    );
+  }
+
+  public componentWillUnmount(): void {
+    clearTimeout(this.timer);
+    clearTimeout(this.increaseTotal);
+  }
+
+  public componentDidUpdate(): void {
+    const percentComplete = this.state.total / this.state.photos.length;
+    if (percentComplete !== this.state.percentComplete) {
+      this.setState({ percentComplete: percentComplete });
+    }
+  }
+
+  private _onToggleChange(): void {
+    this.setState({ loading: !this.state.loading });
+  }
+
+  private _renderAnnounced(): JSX.Element | undefined {
+    const { announced } = this.state;
+
+    return announced;
+  }
+
+  private _createPhotos(): { url: string; width: number; height: number }[] {
+    const result = createArray(30, () => {
+      return {
+        url: `http://placehold.it/100x100`,
+        width: 100,
+        height: 100
+      };
```

Comment:
nit: You may consider making the `width` and `height` constants and string interpolate them in the `url` value to keep them in sync better.

```ts
    const width: number = 100;
    const height: number = 100;

    return {
        url: `http://placehold.it/${width}x${height}`,
        width,
        height
    };
```